### PR TITLE
Add information about WFLY patching into FAQ.

### PR DESCRIPTION
**Pull request by manovotn**
An attempt to add info on WFLY patching to our website. Few notes:
* I chose to add this into FAQ section, feel free to suggest other place(s)
* Anyone has a clue how to "indent" source code? E.g. make is look like a second level indentation.

### CODE REVIEW

File: _faq/13-patch-wfly-with-weld.asciidoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= How to patch Wildfly with newer Weld version?
```
Comment (mkouba) [inline _faq/13-patch-wfly-with-weld.asciidoc]: It's Wild**F**ly :-)

Comment (manovotn) [inline _faq/13-patch-wfly-with-weld.asciidoc]: Yup, thanks.

File: _faq/13-patch-wfly-with-weld.asciidoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Each version of Wildfly is shipped with given version of Weld. However, with every Weld release we also release a patch for Wildfly which allows you to easily upgrade Weld version.
```
Comment (mkouba) [inline _faq/13-patch-wfly-with-weld.asciidoc]: New line before "However".
"However, for every Weld release we also prepare a patch for WildFly..." ?

**Comment by mkouba** [comment]
Looks good now.

**Comment by mkouba** [comment]
Merged, thanks!

**Pull request closed without merging**